src/icons/folderIcons.ts

```diff
@@ -459,7 +459,7 @@ export const folderIcons: FolderTheme[] = [
       },
       { name: 'folder-moon', folderNames: ['.moon'] },
       { name: 'folder-debug', folderNames: ['debug', 'debugging'] },
-      { name: 'folder-fastlane', folderNames: ['fastlane'] },
+      { name: 'folder-fastlane', folderNames: ['fastlane', '.fastlane'] },
```

Comment:
The dot is added per default as a wildcard. See https://github.com/material-extensions/vscode-material-icon-theme/blob/main/CONTRIBUTING.md#folder-icons. So this addition is not required, as the extension will add it automatically.